### DISCUSSION TITLE
Aircraft response discontinuity

### DISCUSSION THREAD

**Discussion by flaviooliva** [JSBSim-Team/jsbsim#556] [Q&A]
Dear friends,
I'm running an elevator pulse simulation and I think the load factor response isn't good because a discontinuity appears that I didn't expect to happen.
Do you think it's a problem? How do you suggest to investigate?
I'm running model 737 in Python.
Thanks.
Flavio Oliv
![Figure 2022-01-04 093259](https://user-images.githubusercontent.com/97100625/148060279-922c088f-d772-495b-abfd-67243a511e47.png)
a

**Comment by seanmcleod** [comment]
@flaviooliva this isn't a problem as such. Think what happens when the elevator is moved, there is a change in the amount of lift generated by the horizontal tail, which impacts the total lift currently being generated by the aircraft. Which you are then measuring via the load factor.

Here is the relevant snippet for the 737.

https://github.com/JSBSim-Team/jsbsim/blob/d9df84593b40a5bddb67a94f51d57cd1e514c313/aircraft/737/737.xml#L691-L699

So with your pulse there is an initial decrease in the total lift, the elevator deflection also generates a pitching moment, which increases alpha, but the increase in alpha takes a while to build given the magnitude of the elevator moment and the aircraft's pitching moment of inertia. But once alpha starts to increase then the total lift increases even with the loss of lift from the horizontal tail and so you then see the load factor increasing.

This effect, i.e. the initial loss of lift as the pilot pulls back on the stick/yoke isn't completely unknown, and it can be an issue in particular during landing when the pilot pulls back to flare in order to reduce the rate of descent for the touchdown. Depending on the aircraft and the exact conditions this can actually result in the aircraft landing with a higher rate of descent than what the aircraft had just before the flare.

For example imagine some of the following combinations.

- Relatively low mass compared to the loss in horizontal tail lift.
- Relatively short moment arm for the horizontal tail for relatively low elevator power
- Relatively large pitching moment of inertia relative to elevator power
- Relatively low Cl gradient

Given that sort of combination the pilot could end up with a relatively significant increase in rate of descent while flaring and if not timed correctly would impact the ground with this higher rate of descent before being able to arrest it via the increased alpha and enough time for the increased alpha to have enough of an effect.
